pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -115,7 +109,7 @@ stdenv.mkDerivation {
     # Binutils, and Apple's "cctools"; "bintools" as an attempt to find an
     # unused middle-ground name that evokes both.
     inherit bintools;
-    inherit libc nativeTools nativeLibc nativePrefix isGNU isClang default_cxx_stdlib_compile;
+    inherit libc nativeTools nativeLibc nativePrefix isGNU isClang;
```

Comment:
Would it make sense to expose the libc++ of the compiler, similar to libc?

Reply:
Yeah, though `null` for `libstdc++` is rather unfortunate. Oh well, all the more reason to get on splitting up the GCC derivation.

Reply:
Perhaps for now `gcc` would make sense? Maybe that could also solve `cc ? gcc`.

Reply:
I'm on the fence about this---I do want to get rid of `cc.gcc` but it's a bit disingenous unless we can somehow filter the other libraries out. Maybe we can leave that for a follow-up PR?

Reply:
https://github.com/NixOS/nixpkgs/pull/91293 got rid of `cc.gcc` here.